src/lib/app/app-icon/app-icon.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
+import { IApp } from '..';
+
+@Component({
+  selector: 'app-app-icon',
+  templateUrl: './app-icon.component.html',
+  styleUrls: ['./app-icon.component.scss'],
+})
+export class VantageAppIconComponent {
+  private _app: IApp;
+  _matListAvatar: boolean;
```

Comment:
Should default to false. And should only be prefixed with an underscore if it is private.